index.html

```diff
@@ -5,12 +5,228 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <style>
+      body {
+      display: flex;
+      flex-direction: column;
+      justify-content: space-between; 
+      align-items: center;
+      margin: 0;
+      font-family: Roboto, sans-serif;
+      min-height: 100vh;
+    }
+
+    .HeadingBox {
+      background-color: #1F2937;
+      padding: 20px;
+      text-align: center;
+      color: #f2f2f2;
+      width: 100vw;
+      flex-direction: row;
+    }
+
+    .HeaderLogo {
+      color: #F9FAF8;
+      font-weight: bold;
+      font-size: 24px;
+      margin-bottom: 10px;
+  }
+
+    .HeaderLink1,
+    .HeaderLink2,
+    .HeaderLink3 {
+     font-size: 18px;
+     color: #E5E7EB
+  }
+
+    .WebsiteInfo {
+      font-size: 48px;
+      font: extra-bold;
+      margin-right: 800px;
+      margin-top: 10px;
+      color: #F9FAF8;
+    }
+
+    .WebsiteSubtext {
+      margin-right: 800px;
+      font-size: 16px;
+      color: #777777;
+    }
+
+    .InfoContainer {
+    display: flex;
+    justify-content: space-between;
+    margin-top: 20px;
+    }
+
+    
+  .SomeRandomInformationBox {
+    border-radius: 10px;
+    padding-top: 20px;
+    display: inline-block;
+    border: solid #2341ec 5px;
+    text-align: center;
+    flex: 1;
+    margin-right: 10px;
+    height: 50px;
+    width:  100px;
+    margin-left: 50px;
+    margin-top: 20px;
+    height: 60px;
+  }
+
+
+
+  .InfoContainer {
+    display: flex;
+  }
+
+    .RandomInfo1,
+    .RandomInfo2,
+    .RandomInfo3,
+    .RandomInfo4 {
+      margin-bottom: 10px;
+    }
+
+    .InspirationalQuoteBox {
+      color: #E5E7EB;
+      width: 1000000px;
+      height: 100px;
+      background-color: #eaeaea;
+      padding: 20px;
+      margin-top: 20px;
+      text-align: center;
+    }
+
+    .Quote {
+      color: #1F2937;
+      font-style: light italic;
+      font-size: 18px;
+      margin-top: 20px;
+      padding-right: 500px;
+      padding-left: 500px;
+    }
+
+    .Author {
+      text-align: right;
+    }
+
+    .SignUpBox {
+      width: 780px;
+      height: 100px;
+      background-color: #3882F6;
+      padding: 23px;
+      margin-top: 23px;
+      text-align: center;
+    }
+
+    .CallToAction {
+      margin-right: 400px;
+      font-weight: bold;
+      font-size: 20px;
+      color: #ffffff;
+      margin-bottom: 10px;
+      background-color: #3882F6;
+    }
+
+
+    .image-placeholder {
+      margin-left: 1000px;
+      margin-top: -30px;
+      width: 300px; 
+      height: 100px;
+      background-color: #969191; 
+      position: relative;
+  }
+
+
+    .SignUpText {
+      margin-right: 186px;
+      margin-top: 10px;
+      color: #ffffff;
+    }
+
+    .SignUpButtonBox {
+      margin-top: 20px;
+    }
+
+    .SignUpButton {
+      background-color: #2b97c9;
+      color: #ffffff;
+      padding: 10px 20px;
+      border-radius: 4px;
+      cursor: pointer;
+    }
+
+    .DownBackground {
```

Comment:
I notice you are naming your css as camel case (like: ExampleName). I think it's more preferred to have it like this:
example-name

So for `DownBackGround`, you would want to have it as `down-background`